Update readme.md and the Tables module docs (#1)
DevExpress/devextreme-quill: README.md

  <a href="https://js.devexpress.com/" title="DevExtreme-Quill">DevExtreme Quill</a>
</h1>

The **DevExtreme Quill** is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.

> **DevExtreme Quill** supports basic table operations and enhances lists rendering. 

Reviewer: ```suggestion
In comparison with the original library, **DevExtreme Quill** supports basic table operations and enhances lists rendering. 
```

> Comparing to the current stable version of Quill, the **DevExtreme Quill** supports basic table operations and brings enhancements to lists rendering. 

Reviewer 2: "Comparing"? Why do you need to use a gerund here?

Reviewer 2: ```suggestion
**DevExtreme Quill** supports basic table operations and enhances lists rendering. 
```

Author: To be honest, I just don't know how to say the same thing without gerund. We need to clarify, why we have our own fork, so I want to compare our fork's functionality with the original Quill.

Reviewer 2: I suggested an alternative version

## Modules

> [Modules](/docs/modules.md) allow users to customize the **DevExtreme Quill**'s behavior and functionality. The officially supported modules are the following:

Reviewer 2: ```suggestion
[Modules](/docs/modules.md) allow users to customize the **DevExtreme Quill**'s behavior and functionality. The following are official modules:
```
